### PR TITLE
unpack local file bundle

### PR DESCRIPTION
**Pull request by santhoshdaivajna**
issue : [317](https://github.com/mudler/luet/issues/317)

take image file path as an input and extract the content to destination path.
ex: luet util unpack file://home/us/alpine.tar /tmp --local

### CODE REVIEW

File: pkg/helpers/docker/docker.go [hunk 1]
```diff
@@ -196,18 +205,26 @@ func DownloadAndExtractDockerImage(ctx luettypes.Context, image, dest string, au
...
 		if err := os.MkdirAll(dest, os.ModePerm); err != nil {
 			return nil, errors.Wrapf(err, "cannot create destination directory")
 		}
 	}
 
-	ref, err := name.ParseReference(local)
-	if err != nil {
-		return nil, err
+	var err error
+	if strings.HasPrefix(local, filePrefix) {
```
Comment (mudler) [inline pkg/helpers/docker/docker.go:216]: This is looking good! I think it's safe to just check the input for the file suffix. No need to specify --local at that point

**Review by mudler — CHANGES_REQUESTED** [review]
Can we just make it available nevertheless if we are specifying --local or not? Local makes sense only to redirect requests to the local daemon

File: pkg/helpers/docker/docker.go (hunk 1 above)
Comment (santhoshdaivajna) [reply pkg/helpers/docker/docker.go:216]: updated.

_DownloadAndExtractDockerImage_ will be called only if it's `!local && !strings.HasPrefix(image, filePrefix)`
rest of the cases - _ExtractDockerImage_ will be called.

**Review by mudler — APPROVED** [review]
looking good! thanks!

**Comment by mudler** [comment]
care to add a small unit test? I'm trying to fix the CI in #319 meanwhile

**Pull request merged**